src/plugins/data/public/field_formats_provider/types.ts

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
Not sure that it's a right place for that file. If I'm not wrong this enum it's a list of registered converters from: 
`src/plugins/data/common/field_formats/converters`

Reply:
Removed